### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by pngwn**
## Description

Please include a concise summary, in clear English, of the changes in this pull request. If it closes an issue, please mention it here.

Closes: #(issue)

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

File: .changeset/clean-rocks-listen.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ---
 "@gradio/app": minor
-"@gradio/cdn-test": minor
```
Comment (pngwn) [inline .changeset/clean-rocks-listen.md:3]: This was causing a CI failure on main.

File: js/_spaces-test/package.json [hunk 2]
```diff
@@ -11,14 +11,7 @@
 		"lint": "prettier --plugin-search-dir . --check .",
 		"format": "prettier --plugin-search-dir . --write ."
 	},
-	"devDependencies": {
```
Comment (pngwn) [inline js/_spaces-test/package.json:14]: I'll deal with this package later, unused atm so this doesn't matter.

File: js/_website/package.json [hunk 3]
```diff
@@ -11,9 +11,9 @@
 		"prepare": "svelte-kit sync"
 	},
 	"devDependencies": {
-		"@sveltejs/adapter-auto": "^2.0.0",
-		"@sveltejs/adapter-static": "^2.0.2",
-		"@sveltejs/kit": "^1.27.6",
```
Comment (pngwn) [inline js/_website/package.json:16]: key updates

File: js/_website/svelte.config.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import adapter from "@sveltejs/adapter-static";
-import { vitePreprocess } from "@sveltejs/kit/vite";
```
Comment (pngwn) [inline js/_website/svelte.config.js:2]: I think this is the only thing we really need to change.

File: package.json [hunk 5]
```diff
@@ -42,7 +42,7 @@
 		"@manypkg/get-packages": "^2.2.0",
 		"@playwright/experimental-ct-svelte": "^1.39.0",
 		"@playwright/test": "^1.39.0",
-		"@sveltejs/vite-plugin-svelte": "^2.5.2",
```
Comment (pngwn) [inline package.json:45]: key update.

File: package.json [hunk 6]
```diff
@@ -84,7 +84,7 @@
 		"tinyspy": "^2.0.0",
 		"typescript": "^5.0.0",
 		"typescript-svelte-plugin": "^0.3.34",
-		"vite": "^4.3.9",
```
Comment (pngwn) [inline package.json:87]: key update.

**Review by aliabd — APPROVED** [review]
Thanks @pngwn! <3 everything looks great!

**Pull request merged**